python2/raygun4py/raygunmsgs.py

```diff
@@ -135,7 +135,10 @@ def __init__(self, exc_type, exc_value, exc_traceback, options):
 
         frames = None
         try:
-            frames = inspect.getinnerframes(exc_traceback)
+            if type(exc_traceback) == list and type(exc_traceback[0]) == tuple and str(exc_traceback[0][0].__class__) == "<type 'frame'>":
+                frames = exc_traceback
+            else:
+                frames = inspect.getinnerframes(exc_traceback)
```

Comment:
From my understanding, `inspect.getinnerframes(exc_traceback)` returns a similar array of frames as `inspect.stack()`. So theoretically, they should be interchangeable without breaking any functionality.